### PR TITLE
Resolves: Quick-preview does not switches to another WI view after the WI title is edited

### PR DESCRIPTION
**Pull request by sahil143**
fixes [osio1655](https://openshift.io/openshiftio/openshiftio/plan/detail/1655)

a

### CODE REVIEW

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2331-1 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2331-fabric8-planner.badger.fabric8.io

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2331-2 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2331-fabric8-planner.badger.fabric8.io

File: src/app/widgets/inlineinput/inlineinput.component.html [hunk 1]
```diff
@@ -8,7 +8,7 @@
   [attr.disabled]="(!onLineClickEdit || readOnly) && !editing ? '' : null"
   [attr.placeholder]="placeholder"
   [class.active]="editing"
-  [innerHTML]="inpvalue"
+  [ngModel]="inpvalue"
```
Comment (nimishamukherjee) [inline src/app/widgets/inlineinput/inlineinput.component.html]: Apply camel case to inpvalue - inpValue or inputValue

Comment (vikram-raj) [reply src/app/widgets/inlineinput/inlineinput.component.html]: @sahil143 Special character does not render as expected after removing ```    innerHTML    ```.

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2331-6 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2331-fabric8-planner.badger.fabric8.io

**Comment by rgarg1** [comment]
@sahil143 @nimishamukherjee Issue is fixed in the snapshot. Approved.

**Comment by fabric8cd** [comment]
@sahil143 fabric8/fabric8-ui:SNAPSHOT-PR-2331-7 is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-2331-fabric8-planner.badger.fabric8.io

**Pull request merged**